README.md

```diff
@@ -310,6 +310,34 @@ clean = sanitizeHtml(dirty, {
 });
 ```
 
+### Allowed CSS Styles
+
+If you wish to allow specific CSS _styles_ on a particular element, you can do that with the `allowedStyles` option. Simply declare your desired attributes as regular expression options within an array for the given attribute. Specific elements will inherit whitelisted attributes from the global (\*) attribute.
+Any other CSS classes are discarded.
+
+This implies that the `style` attribute is allowed on that element
+
+```javascript
+clean = sanitizeHtml(dirty, {
+        allowedTags: ['p'],
+        allowedAttributes: {
+          'p': ["style"],
+        },
+        allowedStyles: {
+          '*': {
+            // Match HEX and RGB
+            'color': [/\#(0x)?[0-9a-f]+/i, /rgb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/],
+            'text-align': [/left/, /right/, /center/],
```

Comment:
since we can represent any of these rules with regexps, we don't need to define the rules as an array, just regexp itself is fine

Reply:
I'm fine either way with how it is, though keeping these as lists could yield a better developer experience

Reply:
I disagree. There are global style whitelistings which should be inherited by individual element whitelistings. Take a look at this example:

```javascript
        allowedStyles: {
          '*': {
            "color": [/yellow/],
            "text-align": [/center/]
          },
          'span': {
            "color": [/green/],
            "font-family": [/helvetica/]
          }
        }
```

This should enable all the properties on this span to pass:

```<span style="color:yellow;text-align:center;font-family:helvetica;"></span>```.

By only using regular expressions in these values we would have to either concatenate regular expressions or put the global and individual regular expressions in an array and check them individually.

Reply:
ahh I see, that's a cool feature